swagger/messaging/activity_message.json

```diff
@@ -0,0 +1,249 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "Activity Message",
+    "description": "A set of API endpoints used by the inTandem platform to manage\n the send a client an activity message",
+    "version": "3.0"
+  },
+  "servers": [
+    {
+      "url": "https://api.vcita.biz/",
+      "description": "API Gateway server"
+    }
+  ],
+  "components": {
+    "securitySchemes": {
+      "Bearer": {
+        "type": "http",
+        "scheme": "bearer"
+      }
+    },
+    "schemas": {
+      "Response": {
```

Comment:
The name response is generic. This is ActivityMessageResponse. The swagger should be readable.